test/main/sentry/index.test.js

```diff
@@ -42,4 +67,40 @@ describe('sentry', () => {
       "{asar}/compiled/main/signers/lattice/Lattice/index"
     ])
   })
+
+  it('should drop events once the rate limit has been reached', () => {
+    initSentry()
+    const sentEvents = mockEvents(Array(10).fill({}))
+
+    expect(sentEvents.slice(0, 5)).toStrictEqual(
+      Array(5).fill(validEvent)
+    )
+    expect(sentEvents.slice(5)).toStrictEqual([null, null, null, null, null])
```

Comment:
another nit but I think the expectation here is that we only report 5 events (not necessarily that we return exactly null) so maybe we can make the first part a little clearer with these expectations:

```suggestion
    // after the limit is reached, this function will return a falsy value rather than the actual event
    const reportedEvents = sentEvents.filter(evt => !!evt)
    expect(reportedEvents).toStrictEqual(Array(5).fill(validEvent))
```